inputs validation in playbooks (#2227)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue causing any command to crash when unable to detect local repository properties.
* Fixed an issue where running in a private gitlab repo caused a warning message to be shown multiple times.
* Added a new validation to the **validate** command to verify that markdown and python files do not contain words related to copyright section.

> * Added  a new validation to the **validate** command to validate all inputs are in use a given playbook

Reviewer: ```suggestion
* Added a new validation to **validate**, making sure all inputs of a playbook are used.
```
